src/richText.js

```diff
@@ -394,7 +394,7 @@ define([
 
         // only support absolute path right now
         if (!form.getMugByPath(xpath) && !/instance\('casedb'\)/.test(xpath)) {
-            return value.replace('<', '&lt;').replace('>', '&gt;');
+            return $('<span>').text(value);
```

Comment:
> Is this supposed to create a bubble?

No, but it should return a jquery object: https://github.com/dimagi/Vellum/blob/a75987b6af73bcd53fe5df36f3c47638ba4301b6/src/richText.js#L388

> What happens if the label is edited and the first 19 characters are deleted

The span is there during the current question being edited. If the question is reloaded the span is gone as this function is only called for xpaths/output values.

This should not be an issue as all interactions that go through ckeditor should be done with getData which will rip out the span before writing to the mugs value.


the reason it was undefined before is here: https://github.com/dimagi/Vellum/blob/a75987b6af73bcd53fe5df36f3c47638ba4301b6/src/richText.js#L428 I could have also had that be able to handle text, but just made this function have a predictable output instead